react-function-components/src/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+function CustomButton(props) {
+  // eslint-disable-next-line react/no-unescaped-entities
+  return <button>"Click Me!"</button>;
```

Comment:
Inside of JSX you don't need quotes around text. Notice in your browser the button text has the quotes around the text, we don't want that.

Reply:
Yea, I was wondering about that - just saw in the instructions it had double quotes I will now remove. Thank you!